9/porting_guide_9.rst

```diff
@@ -8,16 +8,13 @@ Ansible 9 Porting Guide
 =======================
 
 .. contents::
-  :local:
   :depth: 2
 
 
 Ansible 9 is based on Ansible-core 2.16.
 
-
 We suggest you read this page along with the `Ansible 9 Changelog <https://github.com/ansible-community/ansible-build-data/blob/main/9/CHANGELOG-v9.rst>`_ to understand what updates you may need to make.
```

Comment:
Should we change this to point to the .md version instead?

Reply:
Sounds like a good idea to me!